tools/cuds-prioritized-attribution/composer/requirements.txt

```diff
@@ -0,0 +1,5 @@
+apache-airflow[gcp_api]==1.10.5
```

Comment:
This version is not supported by cloud composer.
How are you running this DAG? local/sequential executor airflow? 
If so I think your lack of distributed environment for airflow is hiding a multitude of sins.

Reply:
Done